_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -590,16 +668,17 @@ production-grade VPC on AWS] guide, which includes recommendations for segmentat
 security groups, and remote access. Moreover, our link:https://gruntwork.io/reference-architecture/[Reference
 Architecture] can get you up and running with a secure network configuration immediately.
 
-Recommendations 4.1 and 4.2 forbid allowing ports 22 (SSH) and 3389 (Remote Desktop) from the public Internet
-(`0.0.0.0/0`). This seems entirely reasonable, no? Avoid creating any security groups with these rules. Instead, if you
-require SSH or Remote Desktop to your cloud resources, provide a more restricted CIDR range, such as the IP address of
-your offices.
+Recommendation 5.1 forbits allowing access to the remote server administration ports, such as SSH to port 22 and Remote
+Desktop to port 3389, from the public Internet (`0.0.0.0/0`) on the level of Network ACL rules. Recommendation 5.2 does
+the same for Security Groups.  This seems entirely reasonable, no? Avoid creating any Security Groups and NACL with
```

Comment:
Is this intentionally left here? 😄 
```suggestion
the same for Security Groups.  Avoid creating any Security Groups and NACL with
```

Reply:
This was there in the previous version of the guide, and the context around these recommendations hasn't changed, so I saw no need to reformulate the accompanying text. Happy to incorporate any specific suggestions though!

Reply:
ahh okay! Happy to leave it as it was - I raised it because it seemed like the tone of that specific sentence was a bit different to everything else in this section :)